src/main/java/com/aws/greengrass/mqttbridge/MessageBridge.java

```diff
@@ -64,7 +73,6 @@ public MessageBridge(TopicMapping topicMapping) {
      */
     public void addOrReplaceMessageClient(TopicMapping.TopicType clientType, MessageClient messageClient) {
         messageClientMap.put(clientType, messageClient);
-        updateSubscriptionsForClient(clientType, messageClient);
```

Comment:
These are ordering changes happening. Not against the changes. lgtm. Lets make sure we have the right test coverage to make sure we don't regress anything.

Reply:
sounds good, will double check